src/components/shows/showviewer.module.css

```diff
@@ -0,0 +1,189 @@
+a {
+  color: red;
+}
+
+.aboutImage {
+  object-fit: contain;
+  object-position: center;
+  width: 100%;
+  height: auto;
+}
+
+.aboutImageContainer {
+  margin-top: 30px;
+  margin-left: 48px;
+  display: flex;
+  flex: 0 0 33%;
+}
+
+.aboutText {
+  display: flex;
+  flex-direction: row;
+}
+
+.castImageContainer {
+  display: flex;
+  flex: 0 0 50%;
+}
+
+.castText {
+  margin-left: 24px;
+}
+
+.castText2 {
+  margin-left: 0;
+}
+
+.catchPhrase {
+  font-style: italic;
+  text-align: center;
+  vertical-align: center;
+  margin-bottom: 0px;
+  margin-top: 20px;
+  font-size: large;
+}
+
+.container {
+  margin: 0 auto;
+  padding: 24px 24px;
+  max-width: 960px;
+  background-color: #121212;
+  color: white;
+  align-items: center;
+}
+
+.backButton {
+  display: flex;
+  flex: 0 0 50%;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+.backButton a {
+  color: white;
+  background-color: #cc0000;
+  padding: 10px;
+}
+
+.backButton a:hover {
+  background: rgb(187, 34, 34);
+}
+
+button:hover {
+  opacity: 100%;
+  background: inherit;
+}
+
+.fwrdButton {
+  display: flex;
+  flex: 0 0 50%;
+  flex-direction: row;
+  justify-content: flex-start;
+}
+
+.fwrdButton a {
+  color: white;
+  background-color: #cc0000;
+  padding: 10px;
+}
+
+.fwrdButton a:hover {
+  background: rgb(187, 34, 34);
+}
+
+h3 {
+  margin-bottom: 5px;
+}
+
+.h3alt {
+  margin-top: 25px;
+  margin-bottom: 5px;
+}
+
+.hiddenText1 {
+  color: white;
+  font-size: 16px;
+  font-weight: bold;
+  margin-bottom: 2px;
+}
+
+.hiddenText2 {
+  color: white;
+  font-size: 11px;
+}
+
+.hide {
+  height: 0;
+  display: none;
+}
+
+.imgButton {
+  background: none;
+  border: none;
+  color: inherit;
+  cursor: pointer;
+  width: 225px;
+  height: 260px;
+  transition-duration: 0.75s;
+  overflow: hidden;
+  padding: 10px;
+
+  text-align: center;
+  vertical-align: middle;
+  position: relative;
+
+}
+
+.imgButton:hover .imgView {
+  opacity: 25%;
+}
+
+.imgButton:hover .showText {
+  opacity: 100%;
+}
+
+.imgView {
+  width: 75%;
+  opacity: 100%;
+  transition-duration: 0.75s;
+}
+
+.section {
+  background-color: #262626;
+  color: white;
+}
+
+.showText {
+  opacity: 0%;
+  transition-duration: 0.75s;
+  width: 70%;
+  text-align: center;
+  
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -30%);
+}
+
+.signupButton {
+  text-align: center;
+}
+
+.signupButton a {
+  color: white;
+  background-color: #cc0000;
+  padding: 10px;
+}
+
+.signupButton a:hover {
+  background: rgb(187, 34, 34);
+}
+
+.topButtons {
+  display: flex;
+  flex-direction: row;
+}
+
+.view {
+  max-height: 100%;
+}
```

Comment:
This CSS needs media queries to handle resizing for smaller screens (think mainly phones, but also tablets). A few points:

* At many widths, an awkward gap is created when the screen can't quite fit another column of posters. To resolve this, centre the posters so the gap is distributed either side.
![image](https://user-images.githubusercontent.com/1442037/79943707-b646cc80-84bd-11ea-8962-a0787fee457f.png)

* At single-poster widths, the touch boxes of the posters should ideally fill the entire screen width. This is to avoid users tapping on dead space on mobile.
![image](https://user-images.githubusercontent.com/1442037/79943837-1473af80-84be-11ea-97d1-901aad7ff3d2.png)

* The next and previous show buttons get really awkward when squished into a narrow width. Unsure what the best solution is - shrinking the text would get unreadable really quick, hmm.
![image](https://user-images.githubusercontent.com/1442037/79943885-408f3080-84be-11ea-8584-8e2206de018f.png)

* Percentage layouts break down at smaller widths. Beyond a certain point, you're best off switching to a one-column layout (so it would go quote - poster picture - about the show - the cast - cast picture - sign up). This can be done using a media query to change the flexbox properties.
![image](https://user-images.githubusercontent.com/1442037/79944024-a4b1f480-84be-11ea-9a73-6b2573934a30.png)

* The "Click here to sign up!" button breaks apart when it doesn't have enough width.
![image](https://user-images.githubusercontent.com/1442037/79944110-d88d1a00-84be-11ea-93f7-47be52db13c0.png)

Most of this was discovered by using Chrome Dev tools to test a phone layout, which you can access by pressing F12 and clicking the second icon from the top-left ("Toggle device toolbar" when hovered). I usually test the iPhone X size first, but ideally your website should respond fluidly as you resize it (set to Responsive and then drag the handle on the right hand side of the "window") to ensure it looks great on any phone/tablet/desktop window size! (The smallest width you need to test is 320px, which matches the 2016 and older iPhones).

Reply:
Fixed all bar poster touch boxes, which I don't particularly see to be a problem?